lib/simplecov-html/version.rb

```diff
@@ -1,7 +1,7 @@
 module SimpleCov
   module Formatter
     class HTMLFormatter
-      VERSION = "0.10.1"
+      VERSION = "0.10.1".dup
 
       def VERSION.to_a
```

Comment:
huh, good question. I think I'd have just added a `to_a(string)` method as the easiest fix. Design wise I think an object would be best as it could also hold onto the individual parts internally. But it might be a bit of an overkill for this limited use case :D 

Imo go with whatever you prefer, both are fine by me :) Thanks a lot! :+1: